examples/off_limb_enhance.py

```diff
@@ -52,25 +52,32 @@
 params = np.polyfit(rsun_array[rsun_array < 1.5],
                     np.log(y[rsun_array < 1.5]), 1)
 
+###############################################################################
+# Tell matplotlib to use LaTeX for all the text, and then plot the data and the
+# fit.
+
+plt.rc('text', usetex=True)
 plt.plot(rsun_array, y, label='data')
-label = r'fit=Aexp({:.2f}r)'.format(params[0])
+label = 'fit=$A\exp(${:.2f}$r)$'.format(params[0])
 plt.plot(rsun_array, np.exp(np.poly1d(params)(rsun_array)), label=label)
 plt.yscale('log')
 plt.ylabel('mean DN')
-plt.xlabel('radius r (Rsun)')
+plt.xlabel('radius r ($R_{\odot}$)')
```

Comment:
when using LaTeX specifically `\` it's best to use `r""` raw strings.

Reply:
Like this now?